examples/acl.rs

```diff
@@ -19,15 +19,32 @@ fn get_current_user(ctx: &Context, _args: Vec<ValkeyString>) -> ValkeyResult {
     Ok(ValkeyValue::BulkValkeyString(ctx.get_current_user()))
 }
 
+fn custom_category(_ctx: &Context, _args: Vec<ValkeyString>) -> ValkeyResult { 
+    Ok(().into())
```

Comment:
should we actually do something here?

Reply:
These commands were added just so that we can test on them by adding ACL categories to the commands. Aside from this, they do not serve any purpose.

But @zackcam , to make this a functional command, we should probably at least return an OK response back to the client

Reply:
Updated this to now return an OK response when the command is called